Web/packages/content/src/hooks/useGestureRecognizer.ts

```diff
@@ -81,7 +81,8 @@ export function useGestureRecognizer(
     const g = getClosestGesture();
     onRelease(g);
     patternConstructor.current.clear();
-  }, [onRelease, getClosestGesture]);
+    document?.getSelection()?.removeAllRanges();
```

Comment:
Before you clear selections, could you check `g` is not null?

`releaseHandler` is called on `touchend` and `mouseup`, which means the current code would clear selections even if the gesture does not fire.

https://user-images.githubusercontent.com/10496155/184558213-d7481689-1565-467b-a82c-a9b5224953c9.mov

I believe the correct behavior is to clear selections only when a gesture is recognized. `getClosestGesture` returns `null` if there is no registered gesture that satisfies the pattern similarity threshold, so we do not want to clear selections in that case.

I'm not sure what other things we may add to this function. Putting `removeAllRanges()` inside an IF statement or using the early return pattern should both work fine (at least for now).